compute_tools/src/pg_helpers.rs

```diff
@@ -399,7 +401,10 @@ fn update_pgbouncer_ini(
 /// Tune pgbouncer.
 /// 1. Apply new config using pgbouncer admin console
 /// 2. Add new values to pgbouncer.ini to preserve them after restart
-pub async fn tune_pgbouncer(pgbouncer_config: HashMap<String, String>) -> Result<()> {
+pub async fn tune_pgbouncer(
+    mut pgbouncer_config: IndexMap<String, String>,
```

Comment:
I switched to index map as I was worried the order of TLS settings might matter. Since switching to reload, I think that doesn't matter as they are all set at once. I can revert.